content/en/blog/2022/2022-01-10-meet-our-contributors-APAC-India-region-01.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (India region)"
+date: 2022-01-10
+slug: meet-our-contributors-india
```

Comment:
If this is intended to be part of a series (saw the episode reference down below) I'd include that in the slug like `meet-our-contributors-india-part-1` or something

Reply:
Thanks so much for the review, @mrbobbytables. I've just updated the PR for both the above suggested bits.
- removed the date prefix from the filename
- have updated the slug to `meet-our-contributors-india-ep-01`